### PR TITLE
Update example to 0.6.0

### PR DESCRIPTION
**Pull request by adleong**

### CODE REVIEW

**Comment by esbie** [comment]
⭐ 🍥 🍤

File: dcos/namerd-dcos-gob.yaml [hunk 1]
```diff
@@ -1,20 +1,22 @@
 admin:
   port: 9001
 storage:
-  kind: io.buoyant.namerd.storage.experimental.zk
+  kind: io.l5d.zk
+  experimental: true
   hosts:
   - master.mesos:2181
```
Comment (siggy) [inline dcos/namerd-dcos-gob.yaml]: should this be:
```
zkAddrs:
- host: master.mesos
  port: 2181
```

Comment (adleong) [inline dcos/namerd-dcos-gob.yaml]: yes, good catch

**Comment by siggy** [comment]
👍 ⭐

**Pull request merged**